src/lib_c/x86_64-windows-msvc/c/io.cr

```diff
@@ -15,4 +15,5 @@ lib LibC
   fun _chsize(fd : Int, size : Long) : Int
   fun _get_osfhandle(fd : Int) : IntPtrT
   fun _pipe(pfds : Int*, psize : UInt, textmode : Int) : Int
+  fun fflush(fd : Int) : Int
```

Comment:
`fflush` takes a `FILE*` not an fd. `FlushFileBuffers` is the correct function and takes a `HANDLE`.

Reply:
(Although I messed up and made the `HANDLE` hard to get at here, so perhaps best to leave it unimplemented on windows for now)

Reply:
ok! removing it and raising `NotImplementedError`